lib/internal/bootstrap/node.js

```diff
@@ -231,6 +231,10 @@
 
     setupAllowedFlags();
 
+    if (process.versions.openssl) {
+      NativeModule.require('internal/tls').setupRootCertsFile();
```

Comment:
You could just access the binding directly here if you like

Reply:
I assumed there are codes other than just calling functions from binding before. 

Accessing the binding directly is better as it could indicate why ` if (process.versions.openssl)` is necessary to other contributors.